apps/mobile/screens/auth/ResetPasswordScreen.tsx

```
@@ -0,0 +1,107 @@
+import { Button } from '@/components/ui/Button';
+import { env } from '@/config/env.config';
+import { useLocalSearchParams } from 'expo-router';
+import React from 'react';
+import { Form, Input, Label, Text, YStack } from 'tamagui';
+
+interface Props {}
+
+export const ResetPasswordScreen = ({}: Props) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)